AirSkyBoat/AirSkyBoat
# Summoner Avatar Fixes | SDT Fix

**_I affirm:_**

## What does this pull request do?
- Caps level correction for too weak mobs
- Correct flooring and model ID for Carbuncle calc
- Corrects dINT flooring
- Sets all SDT to 0


## Steps to test these changes
To test Physical BPs:
!changejob smn 15
Summon Carbuncle
Use Poison Nails on a level 1 Wild Rabbit
Damage should return between 46-84

To test Magical BPs:
!changejob smn 27
Summon Ramuh
Use Thunder II on a level 1 Wild Rabbit
Damage should return 105
